community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -409,7 +407,7 @@ object projects:
     // [error] Caused by: java.lang.AssertionError: assertion failed:
     // trait MonadIO has non-class parent: AppliedType(TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class <root>)),module scalaz),Monad),List(TypeRef(ThisType(TypeRef(TermRef(ThisType(TypeRef(NoPrefix,module class scalaz)),module effect),trait MonadIO)),type F)))
 
-    // sbtDocCommand = forceDoc("iterateeJVM"), // Fails with
+    // forceDoc("iterateeJVM"), // Fails with
```

Comment:
why we do need to change that?

Reply:
Ah, I accidentally added this one. It's slightly easier to test the different doc commands if it's like this since it matches the previous paragraph.